BOOTSEQ.md

```diff
@@ -4,77 +4,160 @@
+  - See  _Host Reset or Crash_ and _xPU Reset or Crash_ sections for specifics.
+
+- There are use cases for network boot only where the host is used only to provide power and cooling to one or more xPUs.
 - xPU boot can take some time (full linux distro)
 - There is communication between some of the entities from above
 - More...
 
-## Bootup
+## Simultaneous Host and xPU Bootup
+
+1. Host and xPU are simultaneously powered up
+2. Host and xPU are simultaneously restarted
+
+How tightly coupled should host and xPU be during boot phase?
+
+- This is likely use-case dependent, so OPI should accomodate all cases from tightly coupled to completely independent operation.
+
+Events:-
 
-- Server is Powered On
+- Host is Powered On
 - xPU receives power and starts booting
-  - These events may not be synchronous. The xPU may be installed in a PCIe slot providing persistent power. xPU power cycle may be a function of the host BMC (and potentially the xPU BMC) rather than the power state of the host
-- Should Host OS wait for xPU to finish boot ?
+- Should host OS wait for xPU to finish boot ?
   - For some functions, it must (e.g. NVMe namespace that it needs to boot from)
   - With a pre-provisioned, local host OS present, and for other functions (most notably, networking), the host can continue its boot sequence. Ports should appear down from the host's perspective until the xPU is ready to handle traffic from/to host.
-  - Host OS continues to its boot. xPU needs to either respond to PCIe transactions or respond with retry config cycles(CRS) till it is ready. This automatically holds the Host BIOS. Note that Host BIOS may need to be modified to allow longer CRS response time before timing out.
-- Should xPU OS wait for host to finish boot ?
-  - How tightly coupled should host and xPU be during boot phase?
-- who tells Host OS/xPU OS to wait and how ?
-  - PCIe has a mechanism for config retry cycles(CRS) that can be used.
-- who tells Host OS to to continue / start booting ?
-  - These events may not be synchronous. The xPU may be installed in a PICe slot providing persistent power. xPU power cycle may be a function of the host BMC (and potentially the xPU BMC) rather than the power state of the host
-- is Host CPU halted ?
-- can we wait in UEFI/BIOS in case of network boot via xPU ?
+  - Host OS continues to its boot. xPU needs to either respond to PCIe transactions or respond with retry config cycles (CRS) till it is ready. This automatically holds the host BIOS.
+  - The xPU may hold its PCIe channel down during xPU boot. When the xPU has finished booting, it generates a hot-plug event, and the host will insert the xPU into the PCIe root complex. Is this true for all host OS?
+
+- Who tells host OS/xPU OS to wait and how ?
+  - PCIe has a mechanism for config retry cycles (CRS) that can be used
+  - Host BIOS may need to be modified to allow longer CRS response time before timing out
```

Comment:
@jainvipin.  It is not clear which mechanism you are refering to.
The server BMC could poll the xPU via I2C or NC-SI for its boot status.   To avoid the BMC having to know unique HW locations on xPUs from each curated vendor, the polling mechanism must be abstracted on the xPU. Example Redfish or PLDM.  Again these functions must be functional on the xPU within seconds after power is applied to the xPU.